session.go

```diff
@@ -56,75 +57,49 @@ func (s Session) DefaultFrontpage() ([]*Submission, error) {
 	return submissions, nil
 }
 
-// SubredditSubmissions returns the submissions on the given subreddit.
-func (s Session) SubredditSubmissions(subreddit string) ([]*Submission, error) {
+func (s Session) SortedSubmissions(subreddit string, sort popularitySort, config map[string]string) ([]*Submission, error) {
```

Comment:
I brought my laptop with me today to work so, I was able to update the function.  The whole thing looks like this.

    client := geddit.NewSession("golangtestbotthethird")

	var params = map[string]string{
		"t":     geddit.ThisMonth,
		"limit": "10",
	}
	submissions, _ := client.SortedSubmissions("golang", geddit.TopSubmissions, params)

	for _, submission := range submissions {
		fmt.Println(submission.Title)
	}